packages/react/src/components/radio-button/radio-button.tsx

```diff
@@ -0,0 +1,116 @@
+import { ChangeEvent, FunctionComponent } from 'react';
+import styled from 'styled-components';
+import { focus } from '../../utils/css-state';
+import { useId } from '../../hooks/use-id';
+
+const StyledInput = styled.input<{ disabled?: boolean }>`
+    appearance: none;
+    background-color: ${({ theme, disabled }) => (disabled ? theme.component['radio-button-disabled-background-color'] : theme.component['radio-button-background-color'])};
+    border: 1px solid ${({ theme, disabled }) => (disabled ? theme.component['radio-button-disabled-border-color'] : theme.component['radio-button-border-color'])};
+    border-radius: 50%;
+    color: ${({ theme, disabled }) => (disabled ? theme.component['radio-button-disabled-background-color'] : theme.component['radio-button-background-color'])};
+    display: inline-block;
+    flex-shrink: 0;
+    height: var(--size-1x);
+    margin: 0;
+    margin-top: var(--spacing-half);
+    position: relative;
+    width: var(--size-1x);
+    
+    ${(theme) => focus(theme, { selector: '+' })}
+     
+    &:checked {
+        border: 2px solid ${({ theme, disabled }) => (disabled ? theme.component['radio-button-disabled-border-color'] : theme.component['radio-button-checked-border-color'])};
+        &::after {
+            background-color: ${({ theme, disabled }) => (disabled ? theme.component['radio-button-disabled-border-color'] : theme.component['radio-button-checked-background-color'])};
+            border-radius: 50%;
+            content: '';
+            display: block;
+            height: var(--size-half);
+            left: 50%;
+            position: absolute;
+            top: 50%;
+            transform: translate(-50%, -50%);
+            width: var(--size-half);
+        }
+    }
+        
+    &:hover {
+        border: 1px solid ${({ theme, disabled }) => (disabled ? theme.component['radio-button-disabled-hover-border-color'] : theme.component['radio-button-hover-border-color'])};
+    }
```

Comment:
@maboilard est-ce que le `label` devrait avoir le texte grisé comme dans le figma pour l'état `disabled`? Tu as accepté la PR mais je lève le flag pareil.
Oli, si marc demande la modif, quelques options ou p-e que tu as mieux:
```suggestion
    &:disabled {
        & + label {
            color: [token de couleur];
        }
    }
    
    OU
    
    &:disabled {
        & + ${StyledLabel} {
            color: [token de couleur];
        }
    }
```

Reply:
Tu as bien raison, ce détail m'a échappé. Il faudrait que le token réfère au alias token `color-content-disabled`